tsl/test/expected/gapfill.out

```diff
@@ -144,6 +210,27 @@ FROM (VALUES (1),(2)) v(time)
 GROUP BY 1;
 ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 \set ON_ERROR_STOP 1
+-- test time_bucket_gapfill without aggregation
+-- this will not trigger gapfilling
+SELECT
+  time_bucket_gapfill(1,time,1,11)
+FROM (VALUES (1),(2)) v(time);
+ time_bucket_gapfill 
+---------------------
+                   1
+                   2
+(2 rows)
+
+SELECT
+  time_bucket_gapfill(1,time,1,11),
```

Comment:
again I don't get why this isn't gapfilling

Reply:
because it has no aggregation